libs/ledger-live-common/src/platform/react.ts

```diff
@@ -20,6 +20,10 @@ import {
 import { getParentAccount } from "../account";
 import { listCurrencies } from "../currencies";
 
+/**
+ * TODO: we might want to use "searchParams.append" instead of "searchParams.set"
+ * to handle duplicated query params (example: "?foo=bar&foo=baz")
+ */
```

Comment:
We can also use the `stringify` method of `qs` (https://github.com/ljharb/qs#stringifying)

Reply:
Added as comment 👍